Description: Upstream changes introduced in version 1.9.1-2~precise1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 libgdal-grass (1.9.1-2~precise1) precise; urgency=low
 .
   * Launchpad build for Precise
 .
 The person named in the Author field signed this changelog entry.
Author: Jerome Villeneuve Larouche <jlarouche@mapgears.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- /dev/null
+++ libgdal-grass-1.9.1/VERSION
@@ -0,0 +1 @@
+1.9.1
